compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp

```diff
@@ -308,6 +335,14 @@ TEST_F(ForwardTransposeToAddGraphTest, forward_transpose_add_NEG)
   EXPECT_FALSE(pass.run(_graph.g()));
 }
 
+TEST_F(ForwardTransposeToAddGraphNegTest, forward_transpose_add_non_const_NEG)
+{
+  _graph.init({1, 64, 51, 1}, {0, 3, 2, 1});
+
+  luci::ForwardTransposeOpPass pass;
+  EXPECT_FALSE(pass.run(_graph.g()));
+}
+
```

Comment:
Why is this test added in this PR?

Reply:
Well, for the case where binary op has no const operaters. Is this case going to be covered later?

Reply:
Adding a new test seems not related to reducing duplicate codes.

Is this for increasing the portion of negative test?

Reply:
Ah, sort of. The test is included to prevent minor bugs that may occur during refactoring.

Reply:
I see.